<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>room</class>
 <widget class="QWidget" name="room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能家居-使徒行者</string>
  </property>
  <widget class="QLabel" name="LED1Label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>灯一</string>
   </property>
  </widget>
  <widget class="QLabel" name="LED2Label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>灯二</string>
   </property>
  </widget>
  <widget class="QLabel" name="FanLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>283</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>风扇</string>
   </property>
  </widget>
  <widget class="QLabel" name="DoorLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>门</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LED1PushButton">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>124</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/switch_off.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="LED2PushButton">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>214</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/switch_off.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="FanPushButton">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>324</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/switch_off.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DoorPushButton">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>414</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/switch_off.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="LED1Frame">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/closeled_ico.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="LED2Frame">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>160</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/closeled_ico.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="FanFrame">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>270</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/closefan_ico.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="DoorFrame">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/door/images/door/closedoor_ico.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
